upstairs/src/lib.rs

```diff
@@ -2705,8 +2714,8 @@ impl BlockReqWaiter {
  * A task on the Crucible side will receive a notification that a new
  * operation has landed on the reqs queue and will take action:
  *   Pop the request off the reqs queue.
- *   Copy (and optionally encrypt) any data buffers provided to us by the Guest.
- *   Create one or more downstairs DownstairsIO structures.
+ *   Copy (and optionally encrypt) any data buffers provided to us by the
+ * Guest.   Create one or more downstairs DownstairsIO structures.
```

Comment:
I think this line wrap messed up the indented list

Reply:
addressed in rebase